core/metamod.cpp

```diff
@@ -469,11 +514,26 @@ mm_InitializeGlobals(CreateInterfaceFn engineFactory,
 	provider->Notify_DLLInit_Pre(engineFactory, gamedll_info.factory);
 }
 
-static bool
-Handler_GameInit()
+void
+mm_UnloadMetamod()
+{
+	/* Unload plugins */
+	g_PluginMngr.UnloadAll();
+
+	provider->Notify_DLLShutdown_Pre();
+
+	g_SourceHook.CompleteShutdown();
+}
+
+static void
+mm_HandleGameInit()
 {
 	if (is_game_init)
-		return true;
+		return;
+
+#if SOURCE_ENGINE == SE_DOTA
+	Msg("MMS: GameInit\n");
```

Comment:
These printouts are intentional, but can be relegated to a DevMsg. They're in the same format of existing printouts in the S2 console, of debug ILK and otherwise.